client/src/sass/main.sass

```diff
@@ -288,11 +288,6 @@
 .hidden
   display: none !important
 
-// @bizpal/open-services-bst imports Buefy which uses the .container class name which conflicts with vuetify
-// the following CSS fixes the issues from including the open-services-bst
-#landing-container, .container
-  max-width: unset !important
```

Comment:
I tested locally (in Chrome) with and without this and it's now better without it (ie, responsive).

Reply:
yeah... for what it was, which was basically a questionnaire, it was massive.  it added 15 mb to package by itself.